platform/android/export/export.cpp

```diff
@@ -240,6 +240,9 @@ static const LauncherIcon launcher_adaptive_icon_backgrounds[icon_densities_coun
 	{ "res/mipmap/icon_background.png", 432 }
 };
 
+static const int APK_EXPORT_FORMAT = 0;
+static const int AAB_EXPORT_FORMAT = 1;
```

Comment:
Maybe `EXPORT_FORMAT_APK` and `EXPORT_FORMAT_AAB` would be clearer? Otherwise it sounds like those are unrelated constants which define an export format/version for APK and AAB respectively.

Reply:
Done!